src/panels/lovelace/editor/conditions/types/ha-card-condition-state.ts

```diff
@@ -152,7 +153,7 @@ export class HaCardConditionState extends LitElement {
             "ui.components.entity.entity-state-picker.state"
           )} (${this.hass.localize(
             "ui.panel.lovelace.editor.condition-editor.condition.state.current_state"
-          )}: ${this.hass.formatEntityState(entity)})`;
+          )}: ${getStates(entity).includes(entity.state) ? this.hass.formatEntityState(entity) : entity.state})`;
```

Comment:
Let's remove the current state all together

Reply:
That's a shame to remove, I thought that was a nice feature. 

@piitaya did you have any thoughts?

Reply:
I think it's created confusion for the user. Also, I think it's more an integration problem than a frontend issue. If the state is translated, it should be an enum, right? 

I added it because it was here in the all conditions mode but it's not really consistent with other places. Also, we don't have it in automation and I don't miss it

Reply:
Ok thanks for clarifying. I will change this PR to remove it.

Reply:
> Do we have an example of integration that provides that?

I didn't find one. I saw this come up a couple times in discord and I think it was always for a washing machine, but I never saw what integration it was from. Will keep an eye out in the future.

Yeah I agree an enum would also have fixed this.